lattest-lib/test/Test/Lattest/Util/ModelParsingUtils.hs

```diff
@@ -0,0 +1,36 @@
+{-# LANGUAGE DeriveGeneric #-}
+module Test.Lattest.Util.ModelParsingUtils (
+testReadAutFile
+)
+where
+
+import Prelude hiding (take)
+import Test.HUnit
+import qualified Data.Set as Set
+
+import Lattest.Util.ModelParsingUtils(readAutFile)
+import Lattest.Model.Alphabet(IOAct(..))
+
+expectedTransitions :: [(String, IOAct String String, String)]
+expectedTransitions = 
+    [ ("Idle",         In "coin_i",          "CoinInserted")
+    , ("CoinInserted", In "select_coffee_i", "Brewing")
+    , ("Brewing",      Out "ready_o",        "Ready")
+    , ("Ready",        In "take_cup_i",      "Idle")
+    ]
+
+testReadAutFile :: Test
+testReadAutFile = TestCase $ do
+    let filePath = "./test/Test/Lattest/Util/dummy.aut"
```

Comment:
For me to get this test to pass, I needed to defined this path from the project's root directory (./test/Test/Lattest/Util/dummy.aut) instead of the directory of the unit test file itself (./dummy.aut). @mariabelenrodriguez-utwente, did you run it in a different way, or does this way also work for you? (I run tests via 'stack test')